config/config.go

```diff
@@ -30,7 +30,6 @@ const (
 	ModeSeed      = "seed"
 
 	BlockSyncV0 = "v0"
-	BlockSyncV2 = "v2"
```

Comment:
want to cleanup toml.go as part of this 🥺

Reply:
done!